README.md

```diff
@@ -203,6 +204,7 @@ Asynchronous concurrent distributed networking framework for PHP
 ### HTTP
 
 * [amphp/artax](https://github.com/amphp/artax) - An async HTTP/1.1 client built on [amp](#amphp)
+* [amphp/aerys](https://github.com/amphp/aerys) - A non-blocking HTTP application, WebSocket and file server
```

Comment:
This is [already listed under Web Applications](https://github.com/elazar/asynchronous-php/blame/ca95cc5d900f848f55a8596c310977b1d4a41d61/README.md#L273). If it's listed here, it should be removed from there.